app/Http/Controllers/UserController.php

```diff
@@ -57,26 +56,38 @@ public function login(Request $request): JsonResponse
             $request->session()->regenerate();
             return new JsonResponse();
         }
-        throw ExceptionFactory::unauthorised();
+        Auth::logout();
+        return ExceptionFactory::badCredentials()->render();
     }
 
     #[OA\Get(
-        path: '/api/v1/user/status',
+        path: '/api/v1/user/status/{facility}',
         description: new PermissionDescribe(Permission::unverified, Permission::verified),
         summary: 'User status',
         tags: ['User'],
+        parameters: [
+            new OA\Parameter(
+                name: 'facility',
+                description: 'Facility id',
+                in: 'path',
+                allowEmptyValue: true,
+                schema: new OA\Schema(type: 'string', format: 'uuid', example: ''),
```

Comment:
example `UUID`?

Reply:
tutaj dałem puste, bo to co swagger wyświetla jest niezbyt fajne, chceckbox "send empty value" aktywny tylko jak wartość jest pusta; no i też żeby było mniej klikania jak chcesz dać puste